docs/2/technical/data-format/overview.md

```diff
@@ -80,21 +78,16 @@ To start with something simple we'll take the piece of data we generated with th
 
 #### A) Meta
 
-First of all there is the `meta` part which gives information about the API request itself.
+In the above example, the information was requested without specification of the number of pages or the amount of data per page.  By default, the number of pages (`totalPages`) is set to 1 and the number of data `perPage` is set to 999. Because the example has only one piece of data, it has `one count and one page`. 
 
-Indeed, in the parameters of my request, I had simply asked to have the details without indicating the number of pages of data and data per page. 
-Automatically the number of pages (`totalPage`) is set to its default value of 1 and the number of data `perPage` defaults to 999.
-Out here I only have one piece of data, so I only have `one count and one page`. 
+The `source` object represents where the information came from. In this case, the data payload is from the database or `db`.
 
-Finally the `source` is the database, hence the value `db`.
+Please note that if you have access to the `meta` data but not the `obj` data, this is most likely because of a sharing rule. Please revisit your sharing rule to solve this issue. 
```

Comment:
because you lack the privileges necessary to read it. Your privileges can be adjusted by the creation of a share Rule (link here).